testing/examples/test_boston_housing_meta_optimization.py

```diff
@@ -0,0 +1,10 @@
+from examples.boston_housing_meta_optimization import main
+
+
+def test_boston_housing_meta_optimization():
+    y_train_predicted, y_test_predicted, score_transform, score_test = main()
+
+    assert y_train_predicted is not None
+    assert y_test_predicted is not None
+    assert score_transform is not None
+    assert score_test is not None
```

Comment:
For the reason that the examples are comprehensive as shown to the users, some asserts could be made at the end of each example *IF NEEDED*. This way the example would show the user the expected behavior. 

Note: it's super okay to have examples with no asserts, because if the execution fails, the test will fail and that covers 99% of what we want to test: that it works. Perhaps assert that the score after training is bigger than a certain number would help, too.

Reply:
@guillaume-chevalier this example doesnt work I have problems with it because of the transform function of BaseCrossValidation

Reply:
In the worst case, add this text fixture to skip it if fixing is too hard, but it'd need to work ideally: 

`@pytest.mark.skip(reason="TODO: fix transform function of BaseCrossValidation")`

And open issue. 

I believe that the test should already pass even if the BaseCrossValidation doesn't have a transform function yet because the way that I think the random search is made. Weird.